pkg/minishift/setup/hypervisor/virtualbox_windows.go

```diff
@@ -0,0 +1,66 @@
+/*
+Copyright (C) 2018 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package hypervisor
+
+import (
+	"errors"
+	"fmt"
+	"github.com/minishift/minishift/pkg/util/filehelper"
+	"strings"
+)
+
+var (
+	vbExecutables = []string{"VirtualBox.exe", "VBoxManage.exe"}
+	vbDownloadURL = "https://download.virtualbox.org/virtualbox/5.2.20/VirtualBox-5.2.20-125813-Win.exe"
+)
+
+func detectExistingInstall() (string, error) {
```

Comment:
We already have some of this checks as part of preflight checks

Reply:
@anjannath Sure. Agree, I am working on virtualbox part
and my main motto ATM is not to refactor even if need to duplicate as refactor will touch other features code base and chance of regression is high.

We can create refactor task later on.